webapp/src/pages/profileList/components/ProfileListComponent.js

```diff
@@ -33,15 +33,20 @@ class ProfileListComponent extends Component{
                             });
                     });                
  }
-
+onsub=(user_id)=>{
```

Comment:
Can this method have a more descriptive name please?

Reply:
sorted